### DISCUSSION TITLE
Damage Functions

### DISCUSSION THREAD

**Discussion by josetteaudi** [NHERI-SimCenter/SimCenterCommon#291] [Pelicun] [answered]
Hello,

This is Josette Audi, I was a REU at the SimCenter this summer. I am looking for some guidance on how to use Pelicun to access damage functions.

Thank you!

**Comment by zsarnoczay** [comment, marked as answer]
Hi Josette, 

Thanks for reaching out!

The damage functions that you can find in our [Damage and Loss Model Library](https://github.com/NHERI-SimCenter/DamageAndLossModelLibrary) are bundled with Pelicun, and they are readily available whenever you run an analysis. I can provide more information about how to access those.

If you want to use custom damage functions that are not yet in our DLML, you'll need to provide those in a standard format using CSV files. I can also provide more information about how to do this.

So, please tell me a bit more about what exactly you are trying to do with Pelicun and I'll be able to provide more specific advice.

Thanks,
Adam

**Reply by josetteaudi** [reply]
Hi Adam,

Thank you for the information, I appreciate it. I'm looking to access the HAZUS damage functions used to measure structural damage in coastal A and V flood zones. 

Thanks for your help,
Josette
